app/Models/ApplicationDeploymentQueue.php

```
@@ -85,6 +85,47 @@ public function commitMessage()
+        }
+
+        if ($this->pull_request_id !== 0 && $app->environment_variables_preview) {
+            $lockedVars = $lockedVars->merge(
+                $app->environment_variables_preview
+                    ->where('is_shown_once', true)
+                    ->pluck('real_value', 'key')
+                    ->filter()
+            );
+        }
+
+        foreach ($lockedVars as $key => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)